src/components/AppNavigation/CalendarList/CalendarListNew.vue

```diff
@@ -83,7 +82,7 @@
 				<template #icon>
 					<LinkVariant :size="20" decorative />
 				</template>
-				{{ $t('calendar', 'New subscription from link (read-only)') }}
+				{{ $t('calendar', 'Subscription from link (read-only)') }}
```

Comment:
Why dropping the "New"? Not against it, but might as well drop it from other items in the list ("New calendar").

Reply:
comment from @jancborchardt 
"it might be a bit better to have the main entry say "Add calendar", cause otherwise both the trigger and the first entry are called "New calendar"

And actually the word "New" could be cut from the "Subscription from link" entry, cause it by definition already exists?"